rclc/src/rclc/executor.c

```diff
@@ -958,6 +1357,41 @@ _rclc_take_new_data(rclc_executor_handle_t * handle, rcl_wait_set_t * wait_set)
   return rc;
 }
 
+bool _rclc_check_handle_data_available(rclc_executor_handle_t * handle)
+{
+  switch (handle->type) {
```

Comment:
check argument for NULL pointer

Reply:
Added c0c1ede366250e9ad69c374e0cf4e8cabee673d8